lib/WebpackOptionsApply.js

```diff
@@ -345,6 +346,11 @@ class WebpackOptionsApply extends OptionsApply {
 			new SizeLimitsPlugin(options.performance).apply(compiler);
 		}
 
+		// FIXME(sven): this should be conditional
+		// if (options.optimization.jsIncompatibleExports) {
```

Comment:
`checkIncompatibleWasmExports`

Reply:
What I meant here is that checking for `options.optimization.jsIncompatibleExports` doesn't work but it's probably because it's wrong.